### PR TITLE
Runtimejob API ModelView 

### PR DESCRIPTION
**Pull request by akihikokuroda**
### Summary
This PR adds ModelView API for RuntimeJob model

### Details and comments

With this, the get, list and delete operations are easy and useful but the create operation is quite difficult to issue.  Form the usage of the RuntimeJob view, it may be better to have the following additional API in the `Job`.

`/api/v1/jobs/{job id}/add_runtimejob`
and
`/api/v1/jobs/{job id}/list_runtimejob`

### CODE REVIEW

**Comment by psschwei** [comment]
@akihikokuroda would adding the additional API change this PR? or would it be done separately?

**Comment by akihikokuroda** [comment]
@psschwei I'll do it separately.

**Review by psschwei — COMMENTED** [review]
LGTM
I still feel a bit iffy on the django framework, so I'd like either @IceKhan13 or @Tansito to review also

**Comment by akihikokuroda** [comment]
@psschwei Thanks for review!

File: gateway/api/v1/views.py [hunk 1]
```diff
@@ -68,3 +68,16 @@ class FilesViewSet(views.FilesViewSet):
...
 
     permission_classes = [permissions.IsAuthenticated, IsOwner]
+
+
+class RuntimeJobViewSet(views.RuntimeJobViewSet):  # pylint: disable=too-many-ancestors
+    """
+    RuntimeJob view set first version. Use RuntomeJobSerializer V1.
+    """
+
+    queryset = RuntimeJob.objects.all()
```
Comment (IceKhan13) [inline gateway/api/v1/views.py:78]: it will select all runtime jobs, even not authored by user :)

File: gateway/api/v1/views.py [hunk 2]
```diff
@@ -68,3 +68,16 @@ class FilesViewSet(views.FilesViewSet):
...
+    serializer_class = v1_serializers.RuntimeJobSerializer
+    permission_classes = [permissions.IsAuthenticated]
```
Comment (IceKhan13) [inline gateway/api/v1/views.py:80]: Since `IsOwner` changed, we can add it now here

File: gateway/api/views.py [hunk 3]
```diff
@@ -468,3 +468,17 @@ def upload(self, request):  # pylint: disable=invalid-name
...
+    RuntimeJob ViewSet configuration using ModelViewSet.
+    """
+
+    BASE_NAME = "runtime_jobs"
+
+    def get_serializer_class(self):
+        return self.serializer_class
+
+    def get_queryset(self):
+        return RuntimeJob.objects.all()
```
Comment (IceKhan13) [inline gateway/api/views.py:484]: same as above. We need to select only authored runtime jobs.

**Review by IceKhan13 — COMMENTED** [review]
Looks good. Couple of tweaks and we are good to go!

**Comment by akihikokuroda** [comment]
@IceKhan13 Thanks for review.  I updated the code and also updated the test to make sure it works.

**Comment by akihikokuroda** [comment]
@IceKhan13 WDYT about additional APIs

Adding runtime job to the job:
```
/api/v1/jobs/{job id}/add_runtimejob
```

list the runtime jobs created in the job:
```
/api/v1/jobs/{job id}/list_runtimejob
```

I can create another PR if they are good.

**Comment by IceKhan13** [comment]
> WDYT about additional APIs

sounds like a good idea! Let's create a separate PR for this APIs

Thank you!

**Review by IceKhan13 — APPROVED** [review]
Looks great! Thank you @akihikokuroda !

**Pull request merged**